clients/lsp-fsharp.el

```diff
@@ -176,6 +176,11 @@ available, else the globally installed tool."
   :type 'boolean
   :risky t)
 
+(defcustom lsp-fsharp-workspace-extra-exclude-dirs '()
+  "Additional directories to exclude from FsAutoComplete workspace loading / discovery."
+  :group 'lsp-fsharp
+  :type '(repeat string))
```

Comment:
it should be lsp-string-vector. Why did you change the default?

Reply:
Oh jeeze, I completely lost track of this. Apologies!

I'll get this updated.